tcconfig-schema/src/main/xsd/terracotta.xsd

```diff
@@ -166,7 +178,24 @@
     <xs:attribute name="name" type="xs:string" use="required"/>
     <xs:attribute name="value" type="xs:string" use="required"/>
   </xs:complexType>
-  
+
+  <xs:complexType name="failover-priority">
+    <xs:choice>
+      <xs:element name="consistency" type="tc:consistency" minOccurs="0"/>
+      <xs:element name="availability" type="xs:string" fixed=""/>
+    </xs:choice>
+  </xs:complexType>
+
+  <xs:complexType name="consistency">
+    <xs:all minOccurs="0">
+      <xs:element name="voter" type="tc:voter" minOccurs="0"/>
```

Comment:
There seems to be too many `minOccurs="0"` here

Reply:
Voter tags are really optional. You can define the consistency without defining any voter tags and that's still valid.